packages/cbioportal-frontend-commons/package.json

```diff
@@ -37,13 +37,15 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
+    "@types/numeral": "0.0.22",
```

Comment:
let's remove this one. this is not needed by external projects.

Reply:
Done